Charts.xcodeproj/project.pbxproj

```diff
@@ -1,1235 +1,1235 @@
 // !$*UTF8*$!
 {
-	archiveVersion = 1;
```

Comment:
Is this indent stuff intended fix by you @jjatie or just your Xcode auto-fixed it?

Reply:
Yea, I didn't touch any of that...

Reply:
I double checked in BBEdit, and the xcodeproj file is the same. I don't know why those indents happened.